core/templates/pages/topics-and-skills-dashboard-page/topics-and-skills-dashboard-page.component.html

```diff
@@ -77,7 +77,9 @@
               </div>
               <div ng-if="$ctrl.activeTab === $ctrl.TAB_NAME_SKILLS">
                 <i class="fa fa-angle-left topic-pagination-arrow" ng-click="$ctrl.navigateSkillPage($ctrl.MOVE_TO_PREV_PAGE)"></i>
-                <span class="dashboard-list-pagination-active"><[$ctrl.skillPageNumber+1]></span>
+                <span class="dashboard-list-pagination-value">
+                  <span class="dashboard-list-pagination-active"><[$ctrl.skillPageNumber+1]></span>
```

Comment:
Any specific reason for this change?

Reply:
Yeah, basically wanted to make the pagination values appearance exactly the same b/w topics dashboard and skills dashboard.

Reply:
![image](https://user-images.githubusercontent.com/30312043/89250885-5cfd4600-d633-11ea-88c1-84e3b005950f.png)